examples/3dtiles.html

```diff
@@ -92,34 +92,6 @@
                     n[0].visible = false;
                 }
             }
-
-            // Create a new Layer 3d-tiles For DiscreteLOD
-           // ------------------------------------------------------
-            const $3dTilesLayerDiscreteLOD = new itowns.GeometryLayer('3d-tiles');
-
-            $3dTilesLayerDiscreteLOD.preUpdate = preUpdateGeo;
-            $3dTilesLayerDiscreteLOD.update = itowns.process3dTilesNode(
-                itowns.$3dTilesCulling,
-                itowns.$3dTilesSubdivisionControl
-            );
-            $3dTilesLayerDiscreteLOD.name = 'DiscreteLOD';
-            $3dTilesLayerDiscreteLOD.url = 'http://localhost:8003/tilesets/TilesetWithDiscreteLOD/tileset.json';
-            $3dTilesLayerDiscreteLOD.protocol = '3d-tiles'
-            $3dTilesLayerDiscreteLOD.overrideMaterials = true;  // custom cesium shaders are not functional
-            $3dTilesLayerDiscreteLOD.type = 'geometry';
-            $3dTilesLayerDiscreteLOD.visible = false;
-
-            itowns.View.prototype.addLayer.call(globe, $3dTilesLayerDiscreteLOD);
-
-            // debug for DiscreteLOD
-            const debug3dt = new itowns.GeometryLayer('debug-3d-tiles');
-            debug3dt.type = 'debug';
-            debug3dt.update = updateDebug;
-
-            // for have the same bool for visible
-            debug3dt.visible = $3dTilesLayerDiscreteLOD.visible;
-            itowns.View.prototype.addLayer.call(globe, debug3dt, $3dTilesLayerDiscreteLOD);
-
```

Comment:
Is it possible to have it in another example html if this layer is removed from this example?